config/locales/client.en.yml

```diff
@@ -282,6 +282,7 @@ en:
       moderators: "Moderators"
       stat:
         all_time: "All Time"
+        last_day: "Last 24 hours"
         last_7_days: "Last 7"
```

Comment:
We should add days to "Last 7" and "Last 30", now.

Reply:
yes sure :+1: